packages/react-instantsearch/examples/react-native/.gitignore

```diff
@@ -34,9 +34,20 @@ local.properties
 #
 node_modules/
 npm-debug.log
+yarn-error.log
 
 # BUCK
 buck-out/
 \.buckd/
-android/app/libs
 *.keystore
+
+# fastlane
+#
+# It is recommended to not store the screenshots in the git repo. Instead, use fastlane to re-generate the
+# screenshots whenever they are needed.
+# For more information about the recommended setup visit:
+# https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Gitignore.md
+
+fastlane/report.xml
```

Comment:
All of this files (and same below) are created by the react-native command line. I will probably switch to an expo directory before merging this PR. It's less files, and its easier to have the app running on your phone.